.eslintrc.cjs

```diff
@@ -16,7 +16,10 @@ module.exports = {
   ignorePatterns: [
     ".eslintrc.cjs",
     "jest.config.js",
-    "src/vite-env.d.ts"
+    "src/vite-env.d.ts",
+    // TOFIX: This component has a react/boolean-prop naming problem, because boolean props do not have the symbol at the beginning of the name.   
```

Comment:
```suggestion
    // FIXME: This component has a react/boolean-prop naming problem, because boolean props do not have the symbol at the beginning of the name.   
```

Reply:
IDEs normally recognize FIXME